.spelling

```diff
@@ -1115,6 +1116,7 @@ subdomain
 subdomains
 subnet
 subnets
+subnetwork
 subnetworks
```

Comment:
out of interest, can you remove this if you've added the singular?

Reply:
Looks like the plural is only used in archives anyway.